packages/realm-network-transport/src/DefaultNetworkTransport.ts

```diff
@@ -69,6 +70,7 @@ export class DefaultNetworkTransport implements NetworkTransport {
     try {
       // We'll await the response to catch throw our own error
       return await DefaultNetworkTransport.fetch(url, {
+        ...DefaultNetworkTransport.extraFetchOptions,
```

Comment:
I tried setting this only on the streaming requests, but that yield the following in my test app:

```
 LOG  Running "ReactNativeTestApp" with {"rootTag":61,"initialProps":{}}
 WARN  Invalid responseType: blob
 ERROR  {"code": -1, "message": "undefined is not an object (evaluating 'options.blobId')"}
 WARN  Invalid responseType: blob
 LOG  Unable to symbolicate stack trace: undefined is not an object (evaluating 'options.blobId')
 WARN  Invalid responseType: blob
 WARN  Invalid responseType: blob
 WARN  Invalid responseType: blob
```